derive_client/_clients/rest/http/orders.py

```diff
@@ -76,6 +76,9 @@ def create(
         subaccount_id = self._subaccount.id
         instrument = self._subaccount.markets.get_instrument(instrument_name=instrument_name)
 
+        amount = amount.quantize(instrument.amount_step)
+        limit_price = limit_price.quantize(instrument.tick_size)
```

Comment:
not so sure we should "correct" this. We do now, as per current implementation, tho I would argue against it. What do we favor? Performance, or expedience? If we opt for expedience we dictate! And we should not!